### DISCUSSION TITLE
PB docker can not start after restoring from backup

### DISCUSSION THREAD

**Discussion by lucius100** [pocketbase/pocketbase#3843] [Q&A] [answered]
I tried to restore from backup , after that my container stopped and can not start because of error.

```
2023/11/30 04:24:20 Server started at http://0.0.0.0:8080
├─ REST API: http://0.0.0.0:8080/api/
└─ Admin UI: http://0.0.0.0:8080/_/
2023/11/30 04:34:01 Failed to apply migration 1701318588_created_videos.js: constraint failed: UNIQUE constraint failed: _collections.name (2067)
2023/11/30 04:34:26 Failed to apply migration 1701318588_created_videos.js: constraint failed: UNIQUE constraint failed: _collections.name (2067)
```

I am using docker with dockerfile official guide https://pocketbase.io/docs/going-to-production#using-docker

**Comment by ganigeorgiev** [comment, marked as answer]
The error indicates that the collection that is being created in the "created_videos" migration already exists.

If this collections exists in your docker environment, aka. it was created before with some other migration or manually without any migration at all, and you are trying to sync the migrations from some other project, then the above error is expected to happen.

Note that you can't safely mix&match migrations from different projects. Treat the migrations as part of your source code.
Also if you do schema changes in one environment you need to sync the generated migrations with the other available environments (and eventually resolve conflicts if multiple people were working on the same thing, similar to the other regular git files).
